HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -4168,14 +4126,197 @@ def self.get_default_duct_locations(hpxml)
     return primary_duct_location, secondary_duct_location
   end
 
-  def self.get_default_gshp_pump_power()
-    return 30.0 # W/ton, per ANSI/RESNET/ICC 301-2019 Section 4.4.5 (closed loop)
+  def self.calc_airflow_ratio(airflow_rated, airflow_design, airflow_defect_ratio)
```

Comment:
@jonwinkler This method looks like it's not used. Should it have been used, or did you end up not needing it?

Reply:
@shorowit I ended up not needing it after realizing that calculation had to occur in hvac sizing.

Reply:
Got it. I'll remove the method.